openstack/tools/allocate_vips.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Allocate the last 20 IP addresses, which could be considered for
+# VIPs in OpenStack
+
+source ~/novarc
+num_vips=20
+subnet="subnet_${OS_USERNAME}-psd"
+
+cidr=$(openstack subnet show ${subnet} -c cidr -f value 2>/dev/null)
+
+net_end=$(awk -F'.' '/HostMax/{print $NF}' <<<$(ipcalc -b $cidr))
+
+net_end_suffix=${net_end##*\.}
```

Comment:
This line isn't needed at all. It was already handled by awk.

Likewise you can get rid of [this line](https://github.com/canonical/stsstack-bundles/pull/213/files#diff-24e887a57e2041b3f5464aec5b017164cb68e5d0b70939d342390e6eb276302cR27) from `common/generate_bundle_base`.